saleor/extensions/plugins/avatax/__init__.py

```diff
@@ -150,8 +151,11 @@ def checkout_needs_new_fetch(data, checkout_token: str) -> bool:
 
 
 def taxes_need_new_fetch(data: Dict[str, Any], taxes_token: str) -> bool:
-    """We store the response from avatax. If object doesn't exist in cache or
-    something has changed, then we fetch data from avatax."""
+    """We store the response from avatax.
```

Comment:
I know it was here before but now it looks even more strange with this formatting applied. We could use a docstring like this:

```
"""Check if Avatax's taxes data need to be refetched.

The response from Avatax is stored in a cache. If an object doesn't exist in cache or something has changed, taxes need to be refetched.
"""
```

The docstring for `checkout_needs_new_fetch` should follow the same convention.

Reply:
@korycins Please review this part as well, as you're the author of the original docstrings (I guess).